### PR TITLE
minor additions

### PR DESCRIPTION
**Pull request by rarmatei**
I added a recommendation that students try first to get familiar with the Terminal commands for Git before jumping to a visual client.

~~I also noticed we don't currently use FreeCodeCamp resources, and I thought I introduce it as homework to the first lesson. I think their "challenges" teaching style starts out very simple and covers a lot of the topics we would like students to practice in the beginning. It's a lot of challenges, but might be good to have them gain more experience with the basics.
Please check out the link I posted as homework in lesson 1 and let me know if it's too much, if it should be made optional, or just removed altogether.~~

I removed the FreeCodeCamp homework as @kabaros rightfully pointed out they did during the application process.

### CODE REVIEW

**Comment by rarmatei** [comment]
#5

File: html-css/lesson1.md [hunk 1]
```diff
@@ -158,7 +158,12 @@ You can also set a transparent border, so that it takes up the space without sho
   - Proper use of the semantic HTML tags we discussed.
   - Research the `:first-child` pseudo-class and use it to style the first article so that it stands out from the others.
 
-2. **Optional** (Est. 1 hour) Read about [advanced CSS selectors](http://learn.shayhowe.com/advanced-html-css/complex-selectors/) and then practice by playing this [CSS selector game](https://flukeout.github.io/). It gets hard at the end, but try your best!
+2. (Est. 5 hours) FreeCodeCamp beginner lessons
```
Comment (kabaros) [inline html-css/lesson1.md]: they have already done that freecodecamp challenge as part of the application process.

**Review by kabaros — COMMENTED** [review]
They already do that freecodecamp course as part of the application process.

File: lesson0.md [hunk 2]
```diff
@@ -135,6 +135,7 @@ Follow this tutorial from Github to setup Git https://help.github.com/articles/s
 
 Then [learn how to use git from the Terminal >](https://www.codecademy.com/learn/learn-git)
 
+Try completing [this tutorial >](https://try.github.io/) as well to learn more intermediate features, like branching.
```
Comment (kabaros) [inline lesson0.md:138]: I think this is the same tutorial as the one from codeacademy. The github one just links to codeacademy.

File: html-css/lesson1.md (hunk 1 above)
Comment (rarmatei) [reply html-css/lesson1.md]: Thanks, good to know. Is that the only challenge they've done, then? Can I refer to any other challenges in the future? (like the Bootstrap/responsive one for the second lesson)

**Comment by rarmatei** [comment]
@kabaros I removed the FreeCodeCamp edits. As I also asked directly on your comment, can we use their challenges in future homework assignments? Like the Bootstrap/responsive challenges for the second HTML lesson.

**Comment by kabaros** [comment]
@rarmatei Here is the [application process](https://docs.google.com/document/d/1Z_PHqPPu1I_bEPyLtcMiXNdWs4zDosQsl689av8hqpY/edit?usp=sharing) - they did the Bootstrap one as well

**Review by kabaros — APPROVED** [review]

**Pull request merged**